server/lib/pbench/process_tb.py

```diff
@@ -97,9 +102,10 @@ def process_tb(self) -> Results:
             tb = Path(str(tbmd5)[0 : -len(".md5")])
             tbdir = tb.parent
             controller = tbdir.name
+            agent_dir = Path(self.config.get("pbench-server", "agent-dir"))
```

Comment:
This approach is acceptable if we cannot come up with a better one.  But, having to configure the Server to tell it where the Agent is installed is somewhat disappointing.